src/community/index.md

```diff
@@ -68,35 +68,42 @@ Here’s a few places to join the discussions that help shape the Design System.
 
 Join one of our regular events where we share ideas and work together to solve common problems affecting service teams.
 
-    {%- call imageCard({
-      src: "/community/images/catchup-call-groupshot.jpg",
-      alt: "Catchup call screenshot.",
-      title: "Design systems monthly catchup",
-      large: true
-    }) %}
-      <p>Every month, we host an informal catchup with our community.</p>
-      <p>Hosted on Zoom, the call usually includes a show and tell from the Design System team and contributors, followed by group discussions where everyone can participate.</p>
-      <p><a href="https://mailchi.mp/707ce8dec373/get-updated-by-email-govuk-design-system">Sign up to our mailing list</a> to find out when we announce the next call, so you can register.</p>
-    {% endcall %}
-
-    {%- call imageCard({
-      src: "/community/images/workshop.jpg",
-      alt: "Table with post-it notes, pens and paper.",
-      title: "Join a Workshop",
-      large: true
-    }) %}
-        <p>We often organise workshops to gather ideas and feedback about work we’re doing, particularly when we know it will affect service teams. If you’d like to help the next time we do, <a href="https://mailchi.mp/707ce8dec373/get-updated-by-email-govuk-design-system">sign up to our mailing list</a>.</p>
-    {% endcall %}
-
-    {%- call imageCard({
-      src: "/community/images/dsday23-logo.svg",
-      alt: "Design System Day 2023 Logo.",
-      title: "Design System Day",
-      large: true
-    }) %}
-      <p>Every year, we host an online conference to collaborate and share knowledge about design systems with like-minded people, covering topics like accessibility, community and decision-making.</p>
-      <p>Design System Day 2023 took place on 10 and 11 October. <a href="/community/design-system-day-2023/session-information/">View videos, session information and speaker biographies from the event</a>.</p>
-    {% endcall %}
+{# This div is here because marked gets confused by caller macros and thinks
+that the markup generated by the macro is part of the preceding element #}
```

Comment:
I spent so much time wrestling with this sort of stuff when I updated markdown here a coupla years ago 'cuz a lot of stuff was falling foul of the "Yeah, OK, I'll interpret this as an HTML block" [spec rules](https://github.github.com/gfm/#html-blocks).